### PR TITLE
Fix ktx2 loader error

### PR DESCRIPTION
**Pull request by luzhuang**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/galacean/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Other information:

### CODE REVIEW

File: packages/loader/src/ktx2/KTX2Loader.ts [hunk 1]
```diff
@@ -139,6 +139,9 @@ export class KTX2Loader extends Loader<Texture2D | TextureCube> {
   ): KTX2TargetFormat | null {
     for (let i = 0; i < priorityFormats.length; i++) {
       const capabilities = this._supportedMap[priorityFormats[i]];
+
+      if (!capabilities) return null;
```
Comment (gz65555) [inline packages/loader/src/ktx2/KTX2Loader.ts:143]: what kind of case can make `capabilities` null or undefined?

Comment (GuoLei1990) [inline packages/loader/src/ktx2/KTX2Loader.ts:143]: Do you have case?

Comment (luzhuang) [reply packages/loader/src/ktx2/KTX2Loader.ts:143]: the priorityFormatsValue is [0,4,1,2,3,7]

Comment (luzhuang) [reply packages/loader/src/ktx2/KTX2Loader.ts:143]: it looks like  R8G8B8A8 is not supported

**Comment by GuoLei1990** [comment]
Fixed  in https://github.com/galacean/engine/pull/1829

**Pull request closed without merging**